daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/TestCLISaveParser.scala

```diff
@@ -318,4 +318,18 @@ class TestCLISaveParser {
     }
   }
 
+  @Test def test_CLI_Saving_SaveParser_error(): Unit = {
+    val schema = path(
+      "daffodil-sapi/src/test/resources/test/sapi/mySchema6.dfdl.xsd",
+    )
+
+    withTempFile { parser =>
+      runCLI(args"save-parser -s $schema $parser") { cli =>
+        cli.expectErr("[error]")
+        cli.expectErr("systemId: file:~")
+        cli.expectErr("Schema context: file:~")
```

Comment:
Instead of outputting `file:~/...`, it feels like this wants to be something like:
```
Schema context: daffodil-sapi/src/test/resources/test/sapi/mySchema6.dfdl.xsd",
```
since that is what the user provided. Similar to my other comment about how we want to use the impor/include schemaLocation as a hint of what to display, I think for the root schema we want to display the path that was actually provided.

As another example, if we had foo.jar on DAFFODIL_CLASSPATH and ran something like:
```
daffodil save-parser -s /some/file/in/foo/jar/dfdl.xsd
```
And that resolves to a path in foo.jar, then I think the schema context we output wants to be exactly `/some/file/in/foo/jar/dfdl.xsd`.

So I suggested that based on my other comment, when we resolve a schema, we either want to use the path directly (in the case of importing absolute schemaLocations or locations without an importing schema like this one), or resolve the schemaLocation relative to the importing display location in the case of relative imports with a context.